### PR TITLE
Add Visual Studio Code Prolog extension

### PR DESCRIPTION
**Pull request by XVilka**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -81,6 +81,7 @@
 ## IDE
 
 - [swish](https://github.com/SWI-Prolog/swish) - SWI-Prolog web IDE.
+- [Prolog extension](https://github.com/arthwang/vsc-prolog) for [Visual Studio Code](https://code.visualstudio.com/).
```
Comment (klaussinani) [inline readme.md]: From the [contributing guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md):

> - Use the following format: [package](link) - Description.
> - Additions should be added to the bottom of the relevant category.

Also, something like following would be more fitting: 

- [VSC-Prolog](https://github.com/arthwang/vsc-prolog) -  Prolog extension Visual Studio Code.

**Comment by XVilka** [comment]
Updated, you can now squash and merge.

File: readme.md [hunk 2]
```diff
@@ -82,6 +82,7 @@
 
 - [swish](https://github.com/SWI-Prolog/swish) - SWI-Prolog web IDE.
 - [Prolog Development Tool](http://sewiki.iai.uni-bonn.de/research/pdt/docs/start) - Prolog IDE for Eclipse.
+- [VSC-Prolog](https://github.com/arthwang/vsc-prolog) - Prolog extension for Visual Studio Code..
```
Comment (klaussinani) [inline readme.md]: The trailing semicolon should be removed as well: 

```diff
- Prolog extension for Visual Studio Code..
+ Prolog extension for Visual Studio Code. 
```

File: readme.md (hunk 1 above)
Comment (XVilka) [reply readme.md]: Fixed.

File: readme.md (hunk 2 above)
Comment (XVilka) [reply readme.md]: Thanks, fixed.

**Comment by XVilka** [comment]
Updated.

**Comment by klaussinani** [comment]
Thank you for contributing!

**Pull request merged**